rust/hyperlane-core/src/chain.rs

```diff
@@ -165,6 +167,7 @@ impl HyperlaneDomainProtocol {
             Ethereum => format!("{:?}", H160::from(addr)),
             Fuel => format!("{:?}", addr),
             Sealevel => format!("{:?}", addr),
+            Cosmos => format!("{:?}", addr), // TODO: is this the right format?
```

Comment:
Actually, since you ask, I am wondering if we meant to use `{:x}` formatting. But at least this would be consistent with what we have.